index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <link rel="stylesheet" href="style.css">
+        <title>first mobile app</title>
+    </head>
+
+    <body>
+        <section class="main-page">
```

Comment:
- [ ] **[OPTIONAL]:** There's no need for this page-wrapper, you already have the `<body>` element, an most of the time you don't need to add a lot of global styling here! I would suggest you remove this `<section>`. 🤓 🛠️

Reply:
done